include/operations/blas2_trees.h

```diff
@@ -424,5 +424,44 @@ GerCol<Single, Lower, Diag, Upper, lhs_t, rhs_1_t, rhs_2_t> make_ger_col(
       lhs_, scalar_, rhs_1_, rhs_2_, nWG_row_, nWG_col_, local_memory_size_);
 }
 
+/**** GERP N COLS x (N + 1)/2 ROWS FOR PACKED MATRIX ****/
+/* This is a specialization of the GER class for the packed
+ * symmetric matrices (P stands for Packed in the name). For more details
+ * on matrix layouts, refer to the explanation here:
+ * https://spec.oneapi.io/versions/1.1-rev-1/elements/oneMKL/source/domains/matrix-storage.html#matrix-storage
+ */
+template <bool Single, bool isUpper, typename lhs_t,
```

Comment:
What is it the purpose of `Single` template parameter?

Reply:
The `Single` template parameter is just for distinguishing between `spr` and `spr2` operators. Once we implement the `spr2` operator, we would be able to use the same `Gerp` kernel and use this `Single` template parameter to change the way we perform the computation.

Reply:
Thanks, I understand. 

Maybe, we could rename it to `isRank1` or similar.